src/QuarkusContext.ts

```diff
@@ -67,6 +68,7 @@ export namespace QuarkusContext {
   export function setDefaults(defaults: Defaults): void {
     checkContext();
     context.globalState.update(BUILD_TOOL, defaults.buildTool);
+    context.globalState.update(PLATFORM_VERSION, defaults.platformVersion);
```

Comment:
The whole point of this function is to save the values that were used to the gloablState database so the next time someone uses the wizard, we set the default values to what were used the previous time.

You're saving the last used platform version as the default, but you never retrieve it. The default just ends up being the recommended one. I'm fine with that, but then there's no point in having this line or the one about PLATFORM_VERSION.

Reply:
Should we use the default from https://github.com/redhat-developer/vscode-quarkus/pull/394#discussion_r713196041?

Reply:
No. I wouldn't use a default.